dev-util/cgdb/cgdb-0.6.8.ebuild

```diff
@@ -34,6 +34,11 @@ RDEPEND="
 
 DOCS=( AUTHORS ChangeLog INSTALL NEWS README.md TODO )
 
+PATCHES=(
+	# Bug: #724256
+	"${FILESDIR}/${PN}-env.patch"
```

Comment:
Pop in the version you based on, like: ${PN}-0.6.8-respect-AR.patch (provided it was based on 0.6.8).

Also, could you add some information to the patch on its upstream status and a reference to the bug?

Reply:
Sure, the fix is upstreamted, I took the patch from git, I will add some information about it to the patch file

Reply:
Great!